internal/testing/enginetest/enginetest.go

```diff
@@ -402,7 +431,30 @@ func setupCallTests(t *testing.T, e wasm.Engine) (*wasm.ModuleInstance, wasm.Mod
 	require.NoError(t, err)
 	linkModuleToEngine(imported, importedMe)
 
-	return imported, importedMe
+	// To test stack traces, call the same function from another module
+	importing := &wasm.ModuleInstance{Name: t.Name() + "-importing"}
+
+	// Don't add imported functions yet as NewModuleEngine requires them split.
+	importedFunctions := []*wasm.FunctionInstance{callHostFn}
+
+	// Add the exported function.
+	callImportedHostFn := &wasm.FunctionInstance{
+		Kind:  wasm.FunctionKindWasm,
+		Type:  ft,
+		Body:  []byte{wasm.OpcodeLocalGet, 0, wasm.OpcodeCall, 0 /* only one imported function */, wasm.OpcodeEnd},
+		Index: 1, // after import
+	}
+	addFunction(importing, callImportCallHostFnName, callImportedHostFn)
+
+	// Compile the importing module
+	importingMe, err := e.NewModuleEngine(importing.Name, importedFunctions, importing.Functions, nil, nil)
+	require.NoError(t, err)
+	linkModuleToEngine(importing, importedMe)
```

Comment:
This must be `importingMe`, not `importedMe`.
```suggestion
	linkModuleToEngine(importing, importingMe)
```

Reply:
still failure remains, but this makes the error constsitent between Iterpreter and JIT